api/app/core/schemas/twitter_downloader.py

```diff
@@ -1,16 +1,82 @@
+from typing import List
+
 from pydantic import BaseModel, constr
 
-from app.core.schemas.base import BaseResponse
 from app.v1.constants import RegexPatterns
+from app.core.schemas.camel_case import CamelModel
 
 
 class TwitterVideoDownloaderSchema(BaseModel):
-    """ TODO: add docstring """
-    
+    """
+    Represents the schema for a Twitter video downloader request.
+    Attributes:
+        url (constr): The Twitter video downloader URL.
+    """
+
     url: constr(regex=RegexPatterns.TWITTER_URL_REGEX)
 
 
-class TwitterVideoDownloaderResponse(BaseResponse):
-    """ TODO: add docstring """
+class tweetMedias(CamelModel):
+    """
+    Represents a media object in a tweet.
+    Attributes:
```

Comment:
There should be a empty line before Attributes

Reply:
Refer https://github.com/pizza-society/unicornify/blob/b0c3d93ea4a55a9b17497f77f1860885f4b269aa/api/app/modules/twitter_downloader_mod.py#L12-L20